ui/components/MesheryPerformanceComponent.js

```diff
@@ -117,8 +117,15 @@ class MesheryPerformanceComponent extends React.Component {
 
   handleChange = (name) => (event) => {
     if (name === 'url' && event.target.value !== '') {
-      this.setState({ urlError: false });
+      let urlPattern = event.target.value;
+      let val = urlPattern.match(/(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g);
+      if ( !val )
+        this.setState({ urlError: true });
+      else
```

Comment:
urlError is already `false` in the state, how about we just set `urlError:true` & remove two conditional branches?

Reply:
I added them as these changes are done on the handleChange(), in which the data change is dynamic. So, as soon as the URL is authenticated the error is gone and as the URL is false the error is highlighted. If I remove the else parts then, the above error will occur. 
Also, the outer else is there, so that when the user wipes out the complete URL after typing something wrong in there, at that time the error should be gone. So, the state should be back to false.